iceoryx_posh/include/iceoryx_posh/internal/popo/ports/interface_port_data.hpp

```diff
@@ -31,6 +31,7 @@ struct InterfacePortData : public BasePortData
     InterfacePortData() = default;
     InterfacePortData(const RuntimeName_t& runtimeName, const capro::Interfaces interface) noexcept;
```

Comment:
How is this file related to the design decision document? Doesn't it violate our traceability rule when we change files which are unrelated to the issue?!